homeassistant/components/prosegur/config_flow.py

```diff
@@ -0,0 +1,77 @@
+"""Config flow for Prosegur Alarm integration."""
+import logging
+
+from pyprosegur.auth import COUNTRY, Auth
+from pyprosegur.installation import Installation
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {"username": str, "password": str, "country": vol.In(COUNTRY.keys())}
```

Comment:
Cleanest would be to use `CONF_` constants, see remark above. Are those values not required?

```python3
STEP_USER_DATA_SCHEMA = vol.Schema(
    {
        vol.Required(CONF_USERNAME): str,
        vol.Required(CONF_PASSWORD): str,
        vol.Required(CONF_COUNTRY): vol.In(COUNTRY.keys()),
    }
)
```